frontend/Checkout/components/Totals/index.jsx

```diff
@@ -13,21 +13,25 @@ import * as style from './style';
 const Totals = ({ totals }) => (
   <App>
     {({ checkout }) => (
-      <div className={style.container}>
+      <div className={style.container} data-test-id="total">
```

Comment:
Should be `data-test-id="totals"` because this represents not just one, but a complete collection.